src/psyclone/psyir/nodes/dynamic_omp_task_directive.py

```diff
@@ -94,6 +93,11 @@ class DynamicOMPTaskDirective(OMPTaskDirective):
                           'loop', 'parent_loop']
     )
 
+    _clause_lists = namedtuple(
```

Comment:
You've given this namedtuple the same name as the previous one (I'm slightly surprised Python doesn't complain). Please could you give it a different name and add a brief comment describing what it is for.
Please could you also add a similar comment for _proxy_vars above.

Reply:
Added/fixed